pkg/providers/docker/docker.go

```diff
@@ -163,7 +163,7 @@ func BuildTemplateMap(clusterSpec *cluster.Spec) map[string]interface{} {
 		"corednsRepository":      bundle.KubeDistro.CoreDNS.Repository,
 		"corednsVersion":         bundle.KubeDistro.CoreDNS.Tag,
 		"kindNodeImage":          bundle.EksD.KindNode.VersionedImage(),
-		"extraArgs":              clusterapi.OIDCToExtraArgs(clusterSpec.OIDCConfig).ToPartialYaml(),
+		"oidcExtraArgs":          clusterapi.OIDCToExtraArgs(clusterSpec.OIDCConfig).ToPartialYaml(),
```

Comment:
Could we make this like `apiServerExtraArgs`? or something like that. I know currently we are using it only for oidc but I think it was intended to be extensible.

Reply:
Sure